services/firebase/src/lib/digest.ts

```diff
@@ -0,0 +1,197 @@
+const { createSSRApp } = require('vue');
```

Comment:
use import instead of require

example:
```suggestion
import { createSSRApp } from 'vue'
```

Reply:
formatted with prettier